pages/careers/_human-friendly.html

```diff
@@ -19,15 +19,17 @@ <h3>How We Work</h3>
     <h3>Benefits</h3>
     <p>Our benefits reflect our values. We believe compensation should be fair, transparent, and generous. We hire Grunts in many countries, so some details may vary.</p>
     <ul>
-      <li><strong>Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and location and uses a multiplier to produce a result that's above market for that title and location.</li>
+      <li><strong>Location Independent Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and uses a multiplier to produce a result that's above market for that title. Our salaries are location independent.</li>
       <li><strong>Profit-Sharing Bonus.</strong> We set aside a pot of money at the end of each year based on profits and distribute bonuses according to a formula that uses as inputs your level within the company and the length of your tenure at the company.</li>
-      <li><strong>Performance Bonus.</strong> We give performance-based bonuses as often as once a quarter, depending on your performance.</li>
+      <li><strong>Hardware budget.</strong> We'll buy you a brand new 16" Apple MacBook Pro (or other computer of your choosing of equivalent value) upon joining. It will be owned by you, not the company.</li>
+      <li><strong>Personal Budget.</strong> We'll give you a personal budget of 1000 USD per month to spend on your workspace (e.g., a co-working space), health (e.g., gym, yoga), time (e.g., babysitter), and/or learning (e.g., books, courses).</li>
+    </ul>
+    <p>In addition to the global benefits listed above, we have some US-specific benefits as well:</p>
+    <ul>
       <li><strong>Medical/Dental/Vision Insurance.</strong> We offer a range of high-quality plans with a large portion paid by the company.</li>
```

Comment:
We actually do pay for "premium" health coverage in other countries per https://www.notion.so/gruntwork/Benefits-by-Country-644686d34b4641bf8853754fc49749ff, so could we move this to the "global" section as well (versus the US-specific section)?

Reply:
Ah, that's a good point, I forgot about the extra health coverage! Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/684/commits/0f144aaa4f9c31160c31ca073c20c20a9e483fa4 and I added a sentence to emphasize the extra aspect of this for non-US countries.